chainer/variable.py

```diff
@@ -805,16 +800,6 @@ def cleargrad(self):
         self._grad_var = None
 
     def zerograd(self):
-        """Initializes the gradient array by zeros.
```

Comment:
This is an documented API. https://docs.chainer.org/en/stable/reference/generated/chainer.Variable.html#chainer.Variable.zerograd

How about fixing this as you did in Link?
https://github.com/chainer/chainer/pull/5081/files#diff-349b58823620f69f6f4ad574377a654d